questions/arrays-and-objects.md

```diff
@@ -0,0 +1,29 @@
+# Arrays and Objects
+
+**Q:** How to differentiate between methods that are destructive and non-destructive?
+
+> A: Test by console logging, and act according by declaring and assigning new variables to the non-destructive ones.
```

Comment:
This is a little too general. Think about what a destructive method impacts a given string/array and how a non-destructive method does not.